src/css/our_collection.css

```diff
@@ -33,16 +33,16 @@
   .collection{
     background-image: url(../img/our-collection-png/desktop1x.png);
     background-size: 135px 240px;
-    background-position-y: 275px;
+    background-position-y: 350px;
     padding-top: 70px;
     padding-bottom: 150px;
 }
```

Comment:
@media screen and (min-width: 768px) {
  .collection{
    background-image: url(../img/our-collection-png/tablet1x.png);
    background-position: left 0 top 221px;
    background-size: 114px;
  }
  @media (min-device-pixel-ratio: 2),
  (min-resolution: 192dpi),
  (min-resolution: 2dppx) {
  .collection{
      background-image:url(../img/our-collection-png/tablet2x.png);
      background-size: 114px;
  }
  }
}

@media screen and (min-width: 1280px) {
  .collection{
    background-image: url(../img/our-collection-png/desktop1x.png);
    background-position: left 0 top 221px;
    background-size: 169px; 
}
@media (min-device-pixel-ratio: 2),
(min-resolution: 192dpi),
(min-resolution: 2dppx) {
.collection{
    background-image:url(../img/our-collection-png/desktop2x.png);
    background-size: 169px; 
}
}

}

@media screen and (min-width: 1440px) {
  .collection{
    background-image: url(../img/our-collection-png/desktop1440-1x.png);
    background-position: left 0 top 231px;
}
@media (min-device-pixel-ratio: 2),
(min-resolution: 192dpi),
(min-resolution: 2dppx) {
.collection{
    background-image:url(../img/our-collection-png/desktop1440-2x.png);
    background-size: 169px;
}
}
}